artifactory/utils/pip/setuppy.go

```diff
@@ -43,57 +45,52 @@ func getProjectNameFromFileContent(content []byte) (string, error) {
 // Run egg-info command on setup.py, the command generates metadata files.
 // Return the content of the 'PKG-INFO' file.
 func getEgginfoPkginfoContent(setuppyFilePath, pythonExecutablePath string) ([]byte, error) {
-	tempDirPath, err := fileutils.CreateTempDir()
+	eggBase, err := fileutils.CreateTempDir()
 	if err != nil {
 		return nil, err
 	}
-	defer fileutils.RemoveTempDir(tempDirPath)
+	defer func() {
+		e := fileutils.RemoveTempDir(eggBase)
```

Comment:
will fail on windows without adding @asafgabai's [RemoveTempDir fix](https://github.com/jfrog/jfrog-client-go/commit/c5eca860135e92191a4fc210a5a588acc9887e11#diff-70e8ee9243e1fbe49b6eac290618859aa9a6525ce0f57393982d581ffee4d267) to client-go v1 also.

Reply:
I reverted this change, since https://github.com/jfrog/jfrog-client-go/pull/489 is not planned to be ported to v1